src/Transformer/PrivateStaticTransformer.php

```diff
@@ -102,6 +103,20 @@ protected function getClassConfig($class)
     protected function isConfigProperty(ReflectionProperty $prop)
     {
         if (!$prop->isPrivate()) {
+            // If this non-private static overrides any private configs, make this an error
```

Comment:
I'm going to be really picky and ask if you could flatten this please? Its a bit pyramid-y.

eg.
```
if (!$class->hasProperty($prop->getName()) {
    continue;
}

// etc...
```